src/main/java/org/deepsymmetry/beatlink/AnnouncementSocketConnection.java

```diff
@@ -0,0 +1,294 @@
+package org.deepsymmetry.beatlink;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.*;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class AnnouncementSocketConnection extends LifecycleParticipant implements SocketSender {
+    private static final Logger logger = LoggerFactory.getLogger(AnnouncementSocketConnection.class);
+
+    public static final AnnouncementSocketConnection connectionsManager = new AnnouncementSocketConnection();
```

Comment:
This should be private; you use `getInstance()` if you want the instance.

Reply:
This was from me hacking about trying to get VirtualRekordbox up and running quickly. Thanks for catching.